bedrock/base/templates/macros.html

```diff
@@ -377,6 +377,16 @@
 </header>{#-- /#fxosnav-header --#}
 {%- endmacro %}
 
+{% macro google_play_button(alt_href='', class_name='', extra_attributes=None, id='', target_blank=False) -%}
```

Comment:
Another possible optimization: `target_blank` seems to be used very little, could it just be passed as an `extra_attribute`?